src/v3/src/types/schema.ts

```diff
@@ -385,6 +385,11 @@ export interface RedirectElement extends UISchemaElement {
   options: { url: string; },
 }
 
+export interface HiddenInputElement extends UISchemaElement {
+  type: 'HiddenInput';
+  options: { name: string; value: string | number };
```

Comment:
curious when can `value` be `number`?

Reply:
In its current usage, never 😅  I only added number if there was a use case in the future that needed it, but maybe I shouldn't have done that  😞 . I will remove the number type